Sorting/Sans Pointers/Bubble Sort.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+using namespace std;
+//void BubbleSort(int A[]);
+//int swap (int,int);
+
+int A[50],i,j,n,a,b,t;
```

Comment:
Use only local variables. Remove everything global .

If you see , the array A[50] is actually being remade in main and bubbleSort function , hence the global isnt even of any use.